docs/source/release/v6.5.0/framework.rst

```diff
@@ -16,18 +16,18 @@ Algorithms
+- The algorithm :ref:`FitIncidentSpectrum <algm-FitIncidentSpectrum>` is now able to return the second derivative of the fitted function in addition to the standard outputs of the fitted function and its first derivative.
 
 Bugfixes
 ############
-- Fixed bug where :ref:`ApplyPaalmanPingsCorrection <algm-ApplyPaalmanPingsCorrection>` the input and output workspace names were identical for this algorithm and the child :ref:`Minus <algm-Minus>` algorithm
-- :ref:`SetSample <algm-SetSample>` now accepts numeric inputs of type float, int or string, rather than requiring float.
+- Fixed bug for :ref:`ApplyPaalmanPingsCorrection <algm-ApplyPaalmanPingsCorrection>` when the input and output workspace names were identical for this algorithm and the child :ref:`Minus <algm-Minus>` algorithm.
+- :ref:`SetSample <algm-SetSample>` now accepts numeric inputs of type float, int, or string, rather than requiring float.
 - :ref:`ExtractSpectra <algm-ExtractSpectra>` now checks if the bins flagged as masked are still there after processing, and if they are not, the masking flag is unchecked for that spectrum.
 - Single valued workspaces can now be saved by :ref:`SaveNexusProcessed <algm-SaveNexusProcessed>` and are properly loaded by :ref:`LoadNexusProcessed <algm-LoadNexusProcessed>`.
-- If an event nexus file is loaded using :ref:`LoadEventNexus <algm-LoadEventNexus>` with LoadAllLogs=True any duplicate log entries no longer give an error. Log entries that are nested lower than two levels down are also properly distinguished
-- :ref:`GenerateLogbook <algm-GenerateLogbook>` now does not stop the execution when optional headers are not defined in the relevant Instrument Parameter File but only send a warning that they are not available.
-- Fixed an issue in :ref:`Fit <algm-Fit>`, where a lack of operator between parameter and value caused a hard crash. Now, the input string is validated.
+- If an event nexus file is loaded using :ref:`LoadEventNexus <algm-LoadEventNexus>` with ``LoadAllLogs=True``, any duplicate log entries no longer give an error. Log entries that are nested lower than two levels down are also properly distinguished.
+- :ref:`GenerateLogbook <algm-GenerateLogbook>` now does not stop the execution when optional headers are not defined in the relevant :ref:`Instrument Parameter File <InstrumentParameterFile>`, but only sends a warning that they are not available.
+- Fixed an issue in :ref:`Fit <algm-Fit>` where a lack of operator between parameter and value caused a hard crash. The input string is now validated.
 - Fixed bug in documentation of :ref:`GroupToXResolution <algm-GroupToXResolution>` where the figures showing the algorithm output were all blank.
 - Fix a crash that could happen in :ref:`LoadEventNexus <algm-LoadEventNexus>` if there are no pulse times.
```

Comment:
There is another release note relating to `LoadEventNexus` in this section. Could these be put adjacent to one another so users are aware of all the changes to that algorithm in this release?

Reply:
Pushed a change